src/frontend/src/optimizer/plan_node/batch_project.rs

```diff
@@ -48,6 +49,15 @@ impl BatchProject {
         BatchProject { base, logical }
     }
 
+    pub fn new_with_order(logical: LogicalProject, order: Order) -> Self {
```

Comment:
can we remove this? I think for the BatchLookUp join is ok to give it a order if we Know the order is that **in fact** and the order derivation with its input is a little hard. but for the project its order should be determined by its input :thinking:

Reply:
Yeah, sorry this was my first approach, where I added a projection after the lookup join and add the order injection there. This is of course irrelevant now.